fastlane/lanes/localization.rb

```diff
@@ -168,17 +168,15 @@
   #
   # @option [String|Symbol] app The app to take screenshots for. Must be `wordpress` or `jetpack`
   # @option [String] version The `versionName` of the app, used to extract the release notes from the right GlotPress key. Defaults to current `versionName`.
-  # @option [Int] build_number The `versionCode` of the app, used to save the release notes to the right `changelogs/*.txt` file. Defaults to current `versionCode`.
   # @option [Boolean] skip_release_notes If set to true, will not download release notes. Defaults to `false`. This can be useful when all you want to download
-  #         is screenshots translations and metadata not linked to a specific version (in which case `version` and `build_number` parameters are optional).
+  #         is screenshots translations and metadata not linked to a specific version (in which case the `version` parameters is optional).
   # @option [Boolean] skip_commit If set to true, will skip the `git add`, `git commit` and `git push` operations. Default to false.
   # @option [Boolean] skip_git_push If set to true, will skip the `git push` at the end. Default to false. Inferred to `true` if `skip_commit` is `true`.
   #
   desc 'Downloads translated metadata from GlotPress'
   lane :download_metadata_strings do |options|
```

Comment:
💡  Ideally we should use the occasion of these kind of PRs (that do small localized changes to the parameters used in our existing lanes) to update those lanes to use keywords syntax (`|app:, version:, skip_release_notes: false, skip_commit: false, skip_git_push: false|`) instead of a generic `options` Hash 🙂